api/internal/conf/conf.go

```diff
@@ -19,14 +19,13 @@ package conf
 import (
 	"fmt"
 	"io/ioutil"
-	"log"
 	"os"
 	"path/filepath"
 	"runtime"
 	"strings"
 
+	"github.com/spf13/viper"
```

Comment:
IMHO Why not just use `cobra`.

Reply:
`cobra` only provides command line management, 
`viper` provides configuration file reading and parsing function.

Reply:
Got it.